cypress/integration/login_page.js

```diff
@@ -0,0 +1,20 @@
+describe("Problem Detailed Page", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/");
+    cy.get('a[href*="/moderator"]').click();
+  });
+
+  it("displays error message when inputted email and password didn't pass validation", () => {
+    cy.get('input[name="email"]').type("test");
+    cy.get('input[name="password"]').type("123");
+
+    cy.contains("Błędny email");
+  });
+
+  it("displays error message when inputted password didn't pass validation", () => {
```

Comment:
Oh, now I see, so this sort of validation will be useful when the user is creating a password, but not when logging in, it can be too informative for an attacker. ;)

Reply:
Ok, I removed those conditions :)